src/components/scene/stats.js

```diff
@@ -16,6 +17,9 @@ module.exports.Component = registerComponent('stats', {
 
   init: function () {
     var scene = this.el;
+
+    if (utils.getUrlParameter('stats') === 'false') { return; }
```

Comment:
> Why would there not be a use case for them? I can imagine someone writing a script that fires up a handful of scenes to quickly look at the stats.

That's not really a use case. Because that's not an actual flow I've needed outside of A-Frame nor do I imagine anyone having a handful of scenes that they want to do a stats pass over. Stats are usually used during a deep inspection when you dig down into a single scene, rather than taking a brief look at many. Even then, a console command or coding into the scene is quick enough.

> Just for the sake of expectations, if I see a URL with ?stats=false, I'm going to assume that ?stats=true turns stats on. Why would not enable both?

There's a single very narrow, minor, and non-imagined use case (me sharing other people's scenes that have stats on) that this PR was addressing, and we have a concise one-line solution for it. If someone was clicking a scene on the blog or Twitter, they aren't going to try messing with the URL or wanting to look at the stats.